controllers/databaseclusterbackup_controller.go

```diff
@@ -344,3 +550,17 @@ func (r *DatabaseClusterBackupReconciler) pgRepoName(everestBackup *everestv1alp
 
 	return "", errors.Errorf("can't perform the backup because the requested backup location isn't set in the DatabaseCluster")
 }
+
+func backupStorageName(repoName string, cluster *everestv1alpha1.DatabaseCluster) (string, error) {
+	// repoNames in a PG cluster are in form "repo1", "repo2" etc.
+	// which is mapped to the DatabaseCluster schedules list.
+	// So here we figure out the BackupStorageName of the corresponding schedule.
+	scheduleInd, err := strconv.Atoi(strings.TrimPrefix(repoName, "repo"))
+	if err != nil {
+		return "", errors.New("Unable to get the schedule index")
```

Comment:
Would it make sense to include the `repoName` in the error to make debugging easier in case this fails?